wolfcrypt/src/hmac.c

```diff
@@ -788,40 +788,35 @@ int wolfSSL_GetHmacMaxSize(void)
 
 #ifdef HAVE_HKDF
 
-/* HMAC-KDF with hash type, optional salt and info, return 0 on success */
-int wc_HKDF(int type, const byte* inKey, word32 inKeySz,
-                   const byte* salt,  word32 saltSz,
-                   const byte* info,  word32 infoSz,
-                   byte* out,         word32 outSz)
+#ifndef WOLFSSL_HAVE_MIN
+#define WOLFSSL_HAVE_MIN
+
+    static INLINE word32 min(word32 a, word32 b)
+    {
+        return a > b ? b : a;
+    }
+
+#endif /* WOLFSSL_HAVE_MIN */
```

Comment:
This section for WOLFSSL_HAVE_MIN can be removed. Was part of a previous cleanup and is now in misc.c